### ISSUE TITLE
Compilation error with 4.10

### ISSUE THREAD

**Issue by zoechi** [ocaml/ocaml-lsp#121]
```sh
$ opam install --solver=mccs ocaml-lsp-server

<><> Synchronising pinned packages ><><><><><><><><><><><><><><><><><><><><><><>
[ocaml-lsp-server.~dev] no changes from git+https://github.com/ocaml/ocaml-lsp.git

The following actions will be performed:
  ∗ install ocaml-lsp-server ~dev*

<><> Processing actions <><><><><><><><><><><><><><><><><><><><><><><><><><><><>
[ERROR] The compilation of ocaml-lsp-server failed at "/home/user/.opam/opam-init/hooks/sandbox.sh build dune build -j 7 --root . --ignore-promoted-rules --no-config --profile release ocaml-lsp-server.install".

#=== ERROR while compiling ocaml-lsp-server.~dev ==============================#
# context     2.0.3 | linux/x86_64 |  | pinned(git+https://github.com/ocaml/ocaml-lsp.git#5782d1da)
# path        ~/.opam/4.10.0/.opam-switch/build/ocaml-lsp-server.~dev
# command     ~/.opam/opam-init/hooks/sandbox.sh build dune build -j 7 --root . --ignore-promoted-rules --no-config --profile release ocaml-lsp-server.install
# exit-code   1
# env-file    ~/.opam/log/ocaml-lsp-server-22042-c743ac.env
# output-file ~/.opam/log/ocaml-lsp-server-22042-c743ac.out
### output ###
# File "_build/.dune/default/vendor/merlin/src/ocaml/utils/dune", line 2, characters 13-25:
# 2 | (copy_files# 410/*.ml{,i})
#                  ^^^^^^^^^^^^
# Error: Cannot find directory: vendor/merlin/src/ocaml/utils/410



<><> Error report <><><><><><><><><><><><><><><><><><><><><><><><><><><><><><><>
┌─ The following actions failed
│ λ build ocaml-lsp-server ~dev
└─ 
╶─ No changes have been performed
```

**Comment by zoechi** [comment]
Seemed to have worked with 4.09.0

**Comment by rgrinberg** [comment]
We'll update ocaml-lsp to support 4.10 once the merlin support is out of beta.

**Comment by hartmut27** [comment]
Works with `4.09.1` also

**Comment by imbsky** [comment]
Merlin version 3.3.4 has been released.

**Comment by giltho** [comment]
Fixed by 30c8d92
